### ISSUE TITLE
Sort lock and similar profiles[feature request]

### ISSUE THREAD

**Issue by KIRUYXAN** [Siphalor/mouse-wheelie#239]
**Is your feature request related to a problem? Please describe.**
In my inventory, some things are in certain places and when you press sort, they have to be laid out again, soI propose to add a sort lock in the selected slots. In general, in one of the two mods that this one replaces, in addition to sorting and blocking it in certain slots, there was the creation of profiles (when choosing a profile, things are laid out in certain places).

**Describe the solution you'd like**
Some button (preferably in the form of a UI) that allows you to block sorting in the selected slots, and a profiles, or, as in one mod for backpacks, assign places for items.

**Comment by adrianfurgol** [comment]
Yes, this is a feature I need too! I always have my chestplate/elytra on the upper leftmost slot and my backpack shulker right next to it, and I always destroy this layout while sorting my inventory, which leads to me not sorting my inventory at all. This would be a huge quality of life improvement if we were able to exclude certain slots from being sorted.

**Comment by DEMEMZEA** [comment]
Yes, completely. I'd love to be able to sort only a specific part of my inventory, it'd be very nice
